app/policies/macro_policy.rb

```diff
@@ -0,0 +1,21 @@
+class MacroPolicy < ApplicationPolicy
+  def index?
+    @account_user.administrator? || @account.users.include?(@user)
+  end
+
+  def create?
+    @account_user.administrator? || @account.users.include?(@user)
+  end
+
+  def show?
+    @account_user.administrator? || @account.users.include?(@user)
+  end
+
+  def update?
+    @account_user.administrator? || @account.users.include?(@user)
+  end
+
+  def destroy?
+    @account_user.administrator? || @account.users.include?(@user)
+  end
```

Comment:
We can just do true here if all users can use macros. 
since we ensure that the user has access to the account at our auth layer. we don't need to make another db call.
ref: contact_policy